src/components/legend/_legend_item.scss

```
@@ -1,25 +1,23 @@
 $legendItemVerticalPadding: $echLegendRowGap / 2;
-
 .echLegendItem {
   color: $euiTextColor;
   display: flex;
   flex-wrap: nowrap;
   justify-content: space-between;
   align-items: center;
   position: relative;
-
+  > div:not(.background) {
+    // euiPopover adds a div with height of 19px
+    height: 18px; // prevents color dot from shifting
```

Let's change this to match the `euiFontSizeXS` mixin

```suggestion
    // euiPopover adds a div with height of 19px otherwise
    height: #{$euiFontSizeXS * $euiLineHeight}; // prevents color dot from shifting
```

---

https://github.com/elastic/eui/blob/527028606dc2ff601ea4baff3ee83555e05f3a54/src/global_styling/mixins/_typography.scss#L44-L47

---

thanks! Updated in 6d5a240